src/app/services/iron/i-iron-identity-provider.ts

```diff
@@ -0,0 +1,4 @@
+export interface IIronIdentityProvider {
```

Comment:
Why is this named `i-iron-identity-provider`? What does the `I` stand for?

Reply:
i stands for interface. Not sure what filename convention is.

the convention in most languages is this IIron..., I saw it being used in other npm packages and adopted it because its familiar to me.